### PR TITLE
Add Belfiore code for suppressed countries

### LINKED ISSUE #30: Missing Codice Belfiore for suppressed countries
**Describe the bug**
Some (or all) suppressed states/contries are not present in the `esteri.csv` file

**To Reproduce**
1. Given codice fiscale `TTVNLA61B15Z135B`
2. When CodiceFiscale.isFormatValid
3. Then `Fiscal code TTVNLA61B15Z135B invalid. Reason: not found: Z135` (sry, I'm not sure this is the actual error output from this library or we somehow reformat it.)

**Expected behavior**
`Z135` is the Belfiore code for the URSS until 1991 (or 1992). https://www.codicefiscaleonline.com/codice_inverso.php recognizes the fiscal code as correct.

**Additional context**
I found [this file](https://www.istat.it/it/files//2011/01/Elenco-Paesi-esteri-cessati.zip) from istat (from [this page](https://www.istat.it/it/archivio/6747)) where there is a list of suppressed state.

If that's ok with you I can try to conform it to the `esteri.csv` file format.

### PR DESCRIPTION
**Pull request by Giackgamba**
Add 6 suppressed countries in the 'esteri.csv' file.
Information have been gathered from ISTAT website at:
https://www.istat.it/it/files//2011/01/Elenco-Paesi-esteri-cessati.zip

Some information (like the english name) have been filled manually to
the best of my abilities.

Test added to check that those countries are correcctly resolved when
building the fiscal code

### CODE REVIEW

File: src/test/java/it/kamaladafrica/codicefiscale/Issue30Test.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import org.junit.Test;
+
+public class Issue30Test {
+
+	private final Person PERSON = Person.builder().firstname("NLA").lastname("TTV")
+			.birthDate(LocalDate.of(1961, 2, 15)).isFemale(false)
+			.city(City.builder().name("UNIONE SOVIETICA").prov("EUROPA").belfiore("Z135").build()).build();
+
+	@Test
+	public void test_esteri_codes() {
```
Comment (kamaladafrica) [inline src/test/java/it/kamaladafrica/codicefiscale/Issue30Test.java]: You should check against CityProvider, like this:
```java
CityProvider provider = CityProvider.ofDefault();
assertEquals("UNIONE SOVIETICA", provider.findByBelfiore("Z135").getName());
assertEquals("Z135", provider.findByName("UNIONE SOVIETICA").getBelfiore());
```
You don't need to build a person as well as CodiceFiscale objects.

File: src/test/java/it/kamaladafrica/codicefiscale/Issue30Test.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+		CodiceFiscale cf = CodiceFiscale.of("TTVNLA61B15Z135B");
+
+		assertEquals("TTVNLA61B15Z135B", cf.getValue());
+
+		assertEquals(cf.getValue(), CodiceFiscale.of(PERSON).getValue());
+		assertEquals(cf.getPerson(), PERSON);
+
+		assertEquals(cf.getBelfiore().getValue(), "Z135");
+		assertEquals(cf.getBelfiore().getCity().getName(), "UNIONE SOVIETICA");
+		assertEquals(cf.getBelfiore().getCity().getProv(), "EUROPA");
```
Comment (kamaladafrica) [inline src/test/java/it/kamaladafrica/codicefiscale/Issue30Test.java]: Make sure that expected value is the first argument of `assertEquals`

File: src/main/resources/esteri.csv [hunk 3]
```diff
@@ -224,3 +224,9 @@ T;5;Oceania;50;Oceania;988;Terre australi e antartiche francesi;French Southern
 T;4;America;42;America centro meridionale;992;Isole Turks e Caicos;Turks and Caicos Islands;537;Z519;n.d.;TC;TCA;219;GBR
 T;4;America;42;America centro meridionale;994;Isole Vergini britanniche;British Virgin Islands;539;Z525;n.d.;VG;VGB;219;GBR
 T;5;Oceania;50;Oceania;997;Wallis e Futuna;Wallis and Futuna Islands;734;Z729;n.d.;WF;WLF;215;FRA
+S;3;Asia;;;825;Yemen del Sud;South Yemen;;Z250;;YD;YMD;;
+S;1;Europa;;;805;Repubblica Democratica Tedesca;German Democratic Republic;;Z111;;DD;DDR;;
+S;1;Europa;;;802;Unione Sovietica;Union of Soviet Socialist Republics;;Z135;;SU;SUN;;
+S;1;Europa;;;801;Cecoslovacchia;Czechoslovakia;;Z105;;n.d;CSK;;
+S;1;Europa;;;820;Repubblica Socialista Federale di Jugoslavia;Socialist Federal Republic of Yugoslavia;;Z118;;YU;YUG;;
+S;1;Europa;;;224;Serbia e Montenegro;Serbia and Montenegro;;Z157;;n.d;SCG;;
```
Comment (kamaladafrica) [inline src/main/resources/esteri.csv]: Maybe you can move this lines in theirs own file, something like `esteri-cessati.csv`, and then update the `it.kamaladafrica.codicefiscale.city.impl.CityProviderImpl` to support the new csv.
So csv files can be updated separately without lost the new records.
If you are in trouble with that i can help you, obviously

**Review by kamaladafrica — CHANGES_REQUESTED** [review]

File: src/test/java/it/kamaladafrica/codicefiscale/Issue30Test.java (hunk 2 above)
Comment (Giackgamba) [reply src/test/java/it/kamaladafrica/codicefiscale/Issue30Test.java]: thanks for the suggestion

File: src/main/resources/esteri.csv (hunk 3 above)
Comment (Giackgamba) [reply src/main/resources/esteri.csv]: Done. I'm not a Java dev, hope that what I gobbled together is ok

Comment (kamaladafrica) [reply src/main/resources/esteri.csv]: that's good. I only made a little enhancement on streams, but your job was ok.

**Review by kamaladafrica — APPROVED** [review]

**Pull request merged**